File: Desktop/WpfViewer/VB.NET/WpfViewer/Helper/ViewerHelper.vb
Imports System.IO

Public Class ViewerHelper
	Public Shared Function IsRpx(fileName As FileInfo) As Boolean
		Dim extension As Object = Path.GetExtension(fileName.FullName)
		Return ".rpx".Equals(extension)
	End Function

	Public Shared Function IsRdf(fileName As FileInfo) As Boolean
		Dim extension As Object = Path.GetExtension(fileName.FullName)
		Return ".rdf".Equals(extension)
	End Function

	Public Shared Function IsSnap(fileName As FileInfo) As Boolean
		Dim extension As Object = Path.GetExtension(fileName.FullName)
		Return ".rdlx-snap".Equals(extension)
	End Function
End Class

File: Desktop/WpfViewer/C#/WpfViewer/Helper/ViewerHelper.cs
using System;
using System.IO;

namespace ActiveReports.Samples.WPFViewer
{
	internal static class ViewerHelper
	{
		public static bool IsRpx(FileInfo fileName)
		{
			var extension = Path.GetExtension(fileName.FullName);
			return ".rpx".Equals(extension, StringComparison.InvariantCultureIgnoreCase);
		}

		public static bool IsRdf(FileInfo fileName)
		{
			var extension = Path.GetExtension(fileName.FullName);
			return ".rdf".Equals(extension, StringComparison.InvariantCultureIgnoreCase);
		}
		
		public static bool IsSnap(FileInfo fileName)
		{
			var extension = Path.GetExtension(fileName.FullName);
			return ".rdlx-snap".Equals(extension, StringComparison.InvariantCultureIgnoreCase);
		}
	}
}
